ts/TreeUtil.ts

```diff
@@ -209,6 +213,18 @@ function industryFilter(industry: Industry, tree: Vector): boolean {
             return radiusFilter(industry.location, tree, 35_00); // 35m
         case 'firewooddepot':
             return radiusFilter(industry.location, tree, 15_00); // 15m
+        case 'WaterWell':
+            return radiusFilter(industry.location, tree, 10_00); // 10m
+        case 'SandHouse':
```

Comment:
This was renamed to `Sandhouse` (lowercase h) in a recent update. I've taken the position that I'm only going to support the most recent names that RRO uses to call its industries, because supporting aliases is pretty difficult due the number of integration points that there are with this value. It may also be possible to create a legacy industry name mapping at some point in the future, but for now I recommend that you switch your industries to their new names. You can either delete the industry and re-place it in game, or use the industry editor to change the legacy name to its modern name.